prime-dotnet-webapi/Program.cs

```diff
@@ -40,11 +43,39 @@ public static int Main(string[] args)
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
+                .UseSerilog()
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
+                    webBuilder.UseSentry(o => InitSentry(o));
                     webBuilder.UseStartup<Startup>();
-                })
-                .UseSerilog();
+                });
+
+        private static void InitSentry(SentryAspNetCoreOptions sentryOptions)
+        {
+            if (PrimeEnvironment.IsLocal)
+            {
+                sentryOptions.Dsn = PrimeEnvironment.PrimeSentryKeys.DevEnvDsn;
+                sentryOptions.TracesSampleRate = PrimeEnvironment.PrimeSentryKeys.DevEnvTraceSampleRate;
+            }
+            else if (PrimeEnvironment.IsProduction)
+            {
+                sentryOptions.Dsn = PrimeEnvironment.PrimeSentryKeys.ProdEnvDsn;
+                sentryOptions.TracesSampleRate = PrimeEnvironment.PrimeSentryKeys.ProdEnvTraceSampleRate;
+            }
+            else
+            {
+                sentryOptions.Dsn = PrimeEnvironment.PrimeSentryKeys.TestEnvDsn;
+                sentryOptions.TracesSampleRate = PrimeEnvironment.PrimeSentryKeys.TestEnvTraceSampleRate;

```

Comment:
Environment.IsLocal means running on your local machine, and is not the same thing as deployed in dev. This is assigning test values to our dev and PR deployments.

Reply:
Also, with these as environment variables you won't need this method.